### PR TITLE
Fix extra extends and collapse generated pipeline

### PR DESCRIPTION
**Pull request by KevinFairise2**
### What does this PR do?

Fix an issue with new parsing method.

### CODE REVIEW

**Review by chouetz — APPROVED** [review]

File: flake-finder-gitlab-ci.yml
Comment (chouetz) [inline flake-finder-gitlab-ci.yml:1]: Is this file used in tests somewhere?

Comment (KevinFairise2) [reply flake-finder-gitlab-ci.yml:1]: Oops should not have gitted this one

Comment (FlorentClarret) [reply flake-finder-gitlab-ci.yml:1]: you could gitignore it?

Comment (KevinFairise2) [reply flake-finder-gitlab-ci.yml:1]: Will do !

**Pull request merged**